AppDeployToolkit/AppDeployToolkitExtensions.ps1

```diff
@@ -3973,7 +3996,7 @@ function Get-NxtFileEncoding {
 		[String]
 		$Path,
 		[Parameter()]
-		[ValidateSet("Ascii", "BigEndianUTF32", "Default", "String", "Default", "Unknown", "UTF7", "BigEndianUnicode", "Byte", "Oem", "Unicode", "UTF32", "UTF8")]
+		[ValidateSet("Ascii", "Default", "UTF7", "BigEndianUnicode", "Oem", "Unicode", "UTF32", "UTF8", "UTF8withBOM")]
```

Comment:
You added UTF8withBOM but it is not in the validate set of functions that make use of "Encoding" functions like Add-NxtXmlNode

Reply:
I do not find these functions, references please